### ISSUE TITLE
bug: pagination remains active even though there are no records

### ISSUE THREAD

**Issue by davidberenstein1957** [recognai/rubrix#2055]
**Describe the bug**
If you open a page with pagination on page x, even though the current version of the dataset doesn´t have page 6, a blank screen without records or the option to go back to any other page..

**To Reproduce**
1. Go to a dataset with page 2 via an URL
2. Delete all records from page 2 (via Python)
3. click the refresh button

**Expected behavior**
I would expect to get redirected to page 1.

**Screenshots**
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/25269220/208070620-6fa36ee7-b7ce-4f30-a482-858eda184945.png">

**Environment (please complete the following information):**
 - OS [e.g. iOS]: MacOS
 - Browser [e.g. chrome, safari]: Firefox
 - Argilla Version [e.g. 1.0.0]: 1.1.1
 - ElasticSearch Version [e.g. 7.10.2]: N.A.
 - Docker Image (optional) [e.g. argilla:v1.0.0]: 1.1.1

**Additional context**
N.A.

**Comment by Amelie-V** [comment]
@leiyre @keithCuniah please add an effort record in order to prioritize. Thanks!

**Comment by nataliaElv** [comment]
Hey @davidberenstein1957 ! Is this issue still relevant for Argilla 2.0?

**Comment by davidberenstein1957** [comment]
Yes it is:) Not sure @damianpumar or @leiyre feel this is a UI issue that can be handled easily?

**Comment by damianpumar** [comment]
Hi @davidberenstein1957, I think we could easily fix it, the redirect should work only when the criteria doesn't match any records, right?
Hi @nataliaElv you're right, this issue is not still relevant for version 2.0
